package.json

```diff
@@ -31,9 +31,9 @@
       "require": "./src/index.js",
       "import": "./src/index.js"
     },
-    "./vercel": {
-      "require": "./src/vercel.js",
-      "import": "./src/vercel.js"
+    "./no-fs": {
```

Comment:
Are there tests specifically for no-fs? We'll want to ensure it doesn't break since it's exported.

Reply:
Yep, there are 3 `secrets.build` related tests that consume the generated JS module.

Reply:
Those appear to use `index.js` rather than `no-fs` though. We'll want to test the no-fs import explicitly.

Reply:
See https://github.com/DopplerHQ/gitops-secrets-nodejs/pull/18